sdl-core/src/main/scala/io/smartdatalake/workflow/dataobject/JdbcTableDataObject.scala

```diff
@@ -477,6 +521,10 @@ private[smartdatalake] case class JdbcColumn(name: String, isNameCaseSensitiv: B
   def nameEqualsIgnoreCaseSensitive(name: String): Boolean = {
     this.name.equalsIgnoreCase(name)
   }
+  def getSparkField(schema: StructType): Option[StructField] = {
+    if (isNameCaseSensitiv) schema.find(_.name.toLowerCase == name.toLowerCase)
```

Comment:
isNameCaseSensitive

Reply:
I will remove method getSparkField completely... it is a left-over from an earlier approach.